dtfs-central-api/src/v1/controllers/utilisation-report-service/get-utilisation-report-reconciliation-details-by-id.controller/helpers/get-utilisation-report-reconciliation-details.test.ts

```diff
@@ -68,6 +79,8 @@ describe('get-utilisation-report-reconciliation-details-by-id.controller helpers
       const bankName = 'Test bank';
       when(getBankNameById).calledWith(bankId).mockResolvedValue(bankName);
       when(getKeyingSheetForReportId).calledWith(reportId, []).mockResolvedValue([]);
+      when(getFeeRecordPaymentEntityGroups).calledWith([]).mockReturnValue([]);
+      when(mapToFeeRecordPaymentGroups).calledWith([]).mockResolvedValue([]);
```

Comment:
We want to ensure that the functions are called with the correct parameters. As some of these functions use the return value of the previous function call, it makes sense to have these parts of the test setup near each other so that future developers have an easier time reading the function